app/controllers/concerns/vet360/writeable.rb

```diff
@@ -29,15 +34,15 @@ def invalidate_cache
       VAProfileRedis::Cache.invalidate(@current_user)
     end
 
-    private
-
     def build_record(type, params)
```

Comment:
Should private be moved up to include `build_record` here? It seems it's only called from within the class and not externally

Reply:
Moved back in private